conf/test.config

```diff
@@ -20,6 +20,8 @@ params {
     max_time   = '6.h'
 
     // Input data
+    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
+    // TODO nf-core: Give any required params for the test so that command line flags are not needed
 
     input         = 'https://raw.githubusercontent.com/nf-core/test-datasets/createtaxdb/samplesheets/test.csv'
```

Comment:
```suggestion
    // TODO nf-core: Give any required params for the test so that command line flags are not needed
    input         = params.pipelines_testdata_base_path + 'createtaxdb/samplesheets/test.csv'
```